src/components/Navigation/index.js

```diff
@@ -1012,24 +1015,25 @@ class Navigation extends Component {
           </div>
           )}
         <ReactMapGL
-          latitude={viewport.latitude}
-          longitude={viewport.longitude}
-          zoom={viewport.zoom}
-          bearing={viewport.bearing}
-          pitch={viewport.pitch}
+          initialViewState={{
+            longitude: viewport.longitude,
+            latitude: viewport.latitude,
+            zoom: viewport.zoom,
+          }}
+          onLoad={e => this.geolocateControlRef.current && this.geolocateControlRef.current.trigger()}
           onViewportChange={this.viewportChange}
           onContextMenu={null}
           mapStyle={MAP_STYLE}
           width="100%"
           height="100%"
           onNativeClick={this.focus}
```

Comment:
The map height should increase when the user interacts with it, but this callback doesn't seem to be called anymore

Reply:
Yes, it seems like the height property is now part of mapStyle, but I'm not sure how it translates visually for the navigation map.